matter/efr32/efr32mg12/BRD4161A/autogen/sl_event_handler.c

```diff
@@ -81,6 +81,9 @@ void sl_stack_init(void)
 {
     sl_rail_util_pa_init();
     sl_rail_util_pti_init();
+#if !RS91X_BLE_ENABLE
+     sl_bt_rtos_init();
+#endif
```

Comment:
All of the changes should not be cherry-picked. In matter 1.0 we do not use the gecko sdk freertos implementation.

Reply:
Hi @mkardous-silabs  reverted the commit